presto-main/src/main/java/io/prestosql/operator/PipelineStats.java

```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
         return internalNetworkInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the raw input data size in bytes.
+     *
+     * @return the raw input data size in bytes.
+     */
     @JsonProperty
     public DataSize getRawInputDataSize()
     {
         return rawInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the raw input positions of the PipelineStats.
+     *
+     * @return the raw input positions.
+     */
     @JsonProperty
     public long getRawInputPositions()
     {
         return rawInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the processed input data size in bytes.
+     *
+     * @return the processed input data size in bytes.
+     */
     @JsonProperty
     public DataSize getProcessedInputDataSize()
     {
         return processedInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the processed input positions of the PipelineStats.
+     *
+     * @return the processed input positions.
+     */
     @JsonProperty
     public long getProcessedInputPositions()
     {
         return processedInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link Duration} that contains the physical input read time.
+     *
+     * @return the physical input read time.
+     */
     @JsonProperty
     public Duration getPhysicalInputReadTime()
     {
         return physicalInputReadTime;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the output data size in bytes.
+     *
+     * @return the output data size.
+     */
     @JsonProperty
     public DataSize getOutputDataSize()
     {
         return outputDataSize;
     }
 
+    /**
+     * Returns a property that represents the output positions of the PipelineStats.
```

Comment:
It's not clear to me what is this "output positions". Is any other way to make it clear for the reader?